Wikipedia/Defines/WMFDeprecationMacros.h

```diff
@@ -0,0 +1,19 @@
+//
+//  WMFDeprecationMacros.h
+//  Wikipedia
+//
+//  Created by Brian Gerstle on 11/11/15.
+//  Copyright © 2015 Wikimedia Foundation. All rights reserved.
+//
+
+#ifndef WMFDeprecationMacros_h
+#define WMFDeprecationMacros_h
+
+#if __IPHONE_OS_VERSION_MIN_REQUIRED >= __IPHONE_9_0
+#define WMF_DEPRECATED_WHEN_DEPLOY_AT_LEAST_9 __deprecated_msg("This API was only necessary when deployment target was < 9. Please remove any code being executed when OS is less than 9.")
```

Comment:
ok, i can make this so you can toggle it w/a build setting. agreed removing warnings is something we should do after launch